lib/rucio/core/replica.py

```diff
@@ -804,16 +804,19 @@ def _build_list_replicas_pfn(
         sign_urls: bool,
         signature_lifetime: int,
         client_location: "Dict[str, Any]",
+        *,
         session: "Session",
 ) -> str:
     """
     Generate the PFN for the given scope/name on the rse.
     If needed, sign the PFN url
     If relevant, add the server-side root proxy to te pfn url
     """
-    pfn = list(protocol.lfns2pfns(lfns={'scope': scope.external,
-                                        'name': name,
-                                        'path': path}).values())[0]
+    pfn: str = list(
```

Comment:
I have a strong preference on the original style here. This new style makes it much more difficult to read.